editor/Dockerfile

```diff
@@ -132,8 +138,9 @@ RUN [ `echo $version-$module | grep -v '^\(2018.3\|2018.4\).*-android'` ] && exi
 #=======================================================================================
 # [2019.x/2020.x/2021.x-android] Support usage of installed Android SDKs outside of editor
 #=======================================================================================
-RUN [ `echo $version-$module | grep -vP '^20(?!18).*-android'` ] && exit 0 || : \
-  \
+RUN echo "$version-$module" | grep -q -vP '^20(?!18).*android' \
+  && exit 0 \
+  || : \
```

Comment:
Note that this will currently not work, as one file will overwrite the other when multiple components are installed.

Reply:
Can you explain a bit more? I haven't tried all possible combinations, but quickly reading through them I don't see where they would conflict with each other. You will still install just one version of android so only one of these section should run even when you install e.g. windows-mono and mac-mono together with android.

But it's completely possible that I've overlooked something, I was testing this only with 2020.3.3f1 version and "windows-mono android mac-mono" modules.

Reply:
Almost each conditional section writes to the alias file `unity-editor` and some to `~/.bashrc`.

However with your approach we'll need a smart way to combine some of this logic and prevent overwriting the .bashrc with each module.

Reply:
But each of these sections is for different version and only one of them will be executed for each docker image, right?

Reply:
There is only 1 file being created, but it's being created it multiple places.

On line 68, 128, 185.

The code so far was not made to scale for multiple modules. For example, if you were to use webgl or linux-il2cpp combined with android, the `webgl` and `linux-il2cpp` specific fixes are being undone.

What we need to make this PR merge-able is to find an elegant way account for multiple modules when creating that alias script (and the .bashrc file).

Reply:
See the proposed changes.